src/js/components/award/DetailRow.jsx

```diff
@@ -53,24 +42,23 @@ export default class DetailRow extends React.Component {
     }
 
     render() {
-        let value = null;
-        if (this.props.title === "Primary Place of Performance") {
-            value = this.props.value.split('\n').map((item, key) =>
-                <span key={key}>{item}<br /></span>
-            );
+        const maxChars = SummaryPageHelper.maxDescriptionCharacters;
```

Comment:
Not all fields are expandable; only the award and CFDA descriptions. The overflow/expandable feature should probably only be enabled based on a prop.

Reply:
Added an `overflow` prop for award & CFDA descriptions